war/src/main/webapp/css/simple-page.theme.css

```diff
@@ -42,25 +38,29 @@ html {
   height: 130px;
 }
 
-.simple-page form .submit input:focus {
-  background-color: #2171ce; /*var(--main-bg-color-focus);*/
-  border: solid 1px #2171ce; /*var(--main-bg-color-focus);*/
+.simple-page form .submit input:hover {
+  background-color: var(--btn-primary-bg-focus);
```

Comment:
Focus should be the same as hover but adding an outline. Basically there's never going to be a focus variable modifier, just hover and active. Variable names should be consistent within Jenkins core.